homeworks/dobrikbartaeva/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,94 @@
+class Car:
+    car_count = 0
+    def __init__(self, brand: str, model: str, construction_year:int):
+        self.brand = brand
+        self.model = model
+        self.construction_year = construction_year
+        self.mileage = 0
+        self.fuel_level = 100
+        Car.car_count +=1
+
+    def __str__(self):
+        return f"{self.brand} {self.model} ({self.construction_year})"
+
+    def drive(self, distance: float):
+        average_usage = 0.1 #mert a kiinduló 100 ->%
+        if  distance <=0:
+            raise ValueError("Distance should be a positive number!")
+        try:
+            fuel_used = distance*average_usage
+            max_distance=self.fuel_level/average_usage
+            if fuel_used <= self.fuel_level:
+               self.fuel_level -= fuel_used
+               self.mileage +=distance
+               max_distance-= self.mileage
+               print(f"{self} drove: {distance} km - New mileage: {self.mileage} km - Fuel level:{self.fuel_level}%")
+            else: 
+                print(f"You have fuel only for {max_distance} km.")
```

Comment:
szuper lenne ha itt nem csak kiírnád hogy ennyit mehetne, hanem tényleg menne is annyit (változna a self.mileage és a self.fuel_level)